app/lib/main/sharezone_material_app.dart

```diff
@@ -76,3 +73,16 @@ class SharezoneMaterialApp extends StatelessWidget {
     );
   }
 }
+
+ThemeMode _getThemeMode(ThemeBrightness themeBrightness) {
+  switch (themeBrightness) {
+    case ThemeBrightness.dark:
+      return ThemeMode.dark;
+    case ThemeBrightness.light:
+      return ThemeMode.light;
+    case ThemeBrightness.system:
+      return ThemeMode.system;
+    default:
```

Comment:
Would it make more sense to don't have a `default` value? Therefore, the analyzer would show a warning when add a new value for `ThemeBrithness`

Reply:
Good catch! Yeah of course, was maybe left over from the standard template.